arex-web-core/src/main/java/com/arextest/web/core/business/listener/planfinish/PlanFinishedLinstener.java

```diff
@@ -7,4 +7,9 @@ public interface PlanFinishedLinstener {
 
   String planFinishedAction(String appId, String planId, Integer status);
 
+
+  default boolean supportReCalculate() {
```

Comment:
Why not just define an empty implementation of the recalculation event

Reply:
fixed